code/parachain/frame/ibc/src/light_clients.rs

```diff
@@ -76,19 +84,75 @@ impl tendermint_light_client_verifier::host_functions::HostFunctionsProvider
 
 impl ics07_tendermint::HostFunctionsProvider for HostFunctionsManager {}
 
+pub struct GrandpaHederHashesStorageInstance;
+impl StorageInstance for GrandpaHederHashesStorageInstance {
```

Comment:
why do we need to storage instances for this?

Reply:
because I don't use substrate's storage macro here